bitrise.yml

```diff
@@ -138,7 +142,7 @@ workflows:
             # Check for unhandled errors
             go get -u -v github.com/kisielk/errcheck
             # Go lint
-            go get -u -v github.com/golang/lint/golint
+            go get -u -v golang.org/x/lint/golint
```

Comment:
Steps replacing the used test scripts (no installation is needed)
```
    - go-list:
    - golint:
    - errcheck:
    - go-test:
```

Reply:
Fixed